selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/factory/TestFactoryInitializer.java

```diff
@@ -85,6 +84,22 @@ public TestFactoryBuilder fromTemplate(String template) throws Exception {
     return new TestFactoryBuilder(factoryDto);
   }
 
+  private String getTemplateDirectory(String template) {
+    String templateDirectoryName;
```

Comment:
Can be simplified:
```
private String getTemplateDirectory(String template) {
    String templateDirectoryName = infrastructure.toString().toLowerCase();
    
    if (infrastructure == k8s){
      templateDirectoryName = k8s.toString().toLowerCase();
    }

    return format("/templates/factory/%s/%s", templateDirectoryName, template);
```

Reply:
Yes, it can, but case-statement is more preferable so as it more obvious.